### PR TITLE
HomePage model for new site root

### PR DESCRIPTION
**Pull request by kave**
## Additions

- Homepage Model

## Review

- @kurtw 
- @richaagarwal 

## Testing
- Create a new `HomePage` at root. This is page should only be createable at root
- Change `sites` root to new HomePage
screen shot 2016-02-25 at 11 52 51 am

- Publish page and view new root home page

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by kurtw** [comment]
As long as the move works :+1:

**Comment by richaagarwal** [comment]
@kave I get this error `django.db.utils.OperationalError: (1193, "Unknown system variable 'storage_engine'")` when trying to run migrations or the server.

File: cfgov/cfgov/settings/local.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
         'PASSWORD': os.environ.get('MYSQL_PW', ''),
         'HOST': os.environ.get('MYSQL_HOST', ''),  # empty string == localhost
         'PORT': os.environ.get('MYSQL_PORT', ''),  # empty string == default
+        'OPTIONS': {'init_command': 'SET storage_engine=MYISAM', },
```
Comment (kave) [inline cfgov/cfgov/settings/local.py]: @richaagarwal whoops thanks for the catch this doesn't need to be in here.

**Comment by richaagarwal** [comment]
:+1:

File: cfgov/v1/migrations/0052_auto_20160225_1649.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            name='HomePage',
+            fields=[
+                ('cfgovpage_ptr', models.OneToOneField(parent_link=True, auto_created=True, primary_key=True, serialize=False, to='v1.CFGOVPage')),
+            ],
+            options={
+                'abstract': False,
+            },
+            bases=('v1.cfgovpage',),
+        ),
+        migrations.RemoveField(
```
Comment (kurtw) [inline cfgov/v1/migrations/0052_auto_20160225_1649.py]: Remove this now.

**Pull request merged**